test/include/ask_test.rb

```diff
@@ -217,6 +217,17 @@ def initialize
               client.askDialog
             end
           end
+
+          context "when asking for a 'password' and values don't match" do
+            let(:ask) { BASE_ASK.merge("password" => true) }
+
+            it "shows an error message and try run the dialog again" do
+              expect(Yast::UI).to receive(:QueryWidget).
+                with(Id("0_0_pass2"), :Value).and_return("some-other-thing", response)
+              expect(Yast::Popup).to receive(:Error).with("The two passwords mismatch.")
+              client.askDialog
+            end
+          end
```

Comment:
No, I only added the test to make sure that all  `Id(:pass2)` where covered. But sure I can add another test to improve the coverage.